### ISSUE TITLE
ES Module support

### ISSUE THREAD

**Issue by karlhorky** [Kikobeats/top-user-agents#5]
Hi there, first of all, thanks for this package! Really helpful.

The import fails when using the ES Module imports:

```
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".json" for /node_modules/top-user-agents/index.json
```

Would you consider switching the file to JavaScript until JSON + ESM stabilizes in Node?

## Workarounds

Use `createRequire`:

```js
import { createRequire } from "module";
const require = createRequire(import.meta.url);
const testers_developers = require("path/to/json/your-json-file.json")
```

Or use the `--experimental-json-modules` flag for Node.

Source: https://stackoverflow.com/questions/60205891/import-json-extension-in-es6-node-js-throws-an-error

**Comment by Kikobeats** [comment]
> Would you consider switching the file to JavaScript until ESM stabilizes in Node?

just `module.exports = require('./my.json')` could be enought to fix the issue?

BTW, PR are welcome 🙂

**Comment by karlhorky** [comment]
> just `module.exports = require('./my.json')` could be enought to fix the issue?

Yep, seems to work! Opened #6 for this (also added TS types and a Prettier config)

**Comment by Kikobeats** [comment]
Thanks! shipped at `v1.0.21`
